src/ZoneServer/World/Actors/Monsters/Mob.cs

```diff
@@ -381,47 +394,132 @@ private void DropItems(Character killer)
 
 			var rnd = RandomProvider.Get();
 			var autoloot = killer?.Variables.Temp.Get("Autoloot", 0) ?? 0;
+			var worldConf = ZoneServer.Instance.Conf.World;
 
-			foreach (var dropItemData in this.Data.Drops)
-			{
-				var dropChance = GetAdjustedDropRate(dropItemData);
+			var iterations = 1;
 
-				var dropSuccess = rnd.NextDouble() < dropChance / 100f;
-				if (!dropSuccess)
-					continue;
+			// these should never be simultaneously applied, but just in case we apply gold last
```

Comment:
If you do something for a specific reason you might want to not only mention that you're doing it, but also why, because someone else, or you in 2 months time, might not be able to understand it without actually reading the code.

Reply:
I added a bit more detail to that comment (the reason we check gold last is because gold gives the most rolls.